post-processor/yandex-export/post-processor.go

```diff
@@ -187,6 +231,8 @@ func (p *PostProcessor) PostProcess(ctx context.Context, ui packer.Ui, artifact
 	state.Put("driver", driver)
 	state.Put("sdk", driver.SDK())
 	state.Put("ui", ui)
+	// no used
+	gd := &builder.GeneratedData{State: state}
```

Comment:
Since this variable is not used I suggest that it be removed entirely and that the initialization of builder.GenerateData be moved to where it is needed.

Reply:
Good point! Thx!